sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawPort.java

```diff
@@ -28,6 +31,10 @@ public String getEnvKey() {
         return envKey;
     }
 
+    public boolean isAdvertise() {
```

Comment:
Yeah I was leaning towards java convention here. Maybe isAdvertised to make it less cringe-worthy